src/apps/explorer/components/OrderWidget/OrderDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons'
+
+import { RawOrder } from 'api/operator'
+
+import { SimpleTable } from 'components/common/SimpleTable'
+
+const COLUMN_COLOR = '#252535'
+
+const Table = styled(SimpleTable)`
+  > tbody > tr {
+    grid-template-columns: minmax(10rem, 25rem) auto;
+
+    background: ${({ theme }): string => theme.bg2};
+    border-right: 0.1em solid;
+    border-color: ${COLUMN_COLOR};
```

Comment:
the main reason to use theme here is in case we switch themes. there will be mismatch